### PR TITLE
Add CUDA 11.8

### PR DESCRIPTION
**Pull request by m3vaz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by m3vaz** [comment]
@conda-forge-admin, please rerender

File: recipe/install_cuda.bat
Comment (jaimergp) [inline recipe/install_cuda.bat:27]:
```suggestion
if "%CUDA_VERSION%" == "11.8" goto cuda118
```

File: recipe/install_cuda.bat
Comment (jaimergp) [inline recipe/install_cuda.bat:173]:
```suggestion
:cuda118
```

**Review by kkraus14 — APPROVED** [review]

**Review by jaimergp — APPROVED** [review]
LGTM. I'd wait before merge given the current Azure incidence.

**Review by beckermr — CHANGES_REQUESTED** [review]
Please do not merge this pr yet.

**Comment by m3vaz** [comment]
@beckermr Can we merge this yet?

**Comment by beckermr** [comment]
No do not merge.

**Comment by m3vaz** [comment]
From internal discussion, closing until further notice.

**Pull request closed without merging**
